src/components/expansion/ExpansionPanel.svelte

```diff
@@ -237,20 +238,20 @@
         {/if}
       </svelte:fragment>
     </ActivityFilterBuilder>
-    <div class="sne-controls">
+    <div class="sne-controls flex flex-wrap">
       <div class="sne-filter">
         <input
           bind:value={filterText}
-          class="st-input"
+          class="st-input w-32"
           name="search"
           autocomplete="off"
           placeholder="Filter..."
           aria-label="Filter..."
         />
       </div>
       {#if SEQUENCE_EXPANSION_MODE === SequencingMode.TYPESCRIPT}
-        <div class="sne-expansion-set-select">
-          <select name="expansionSetId" bind:value={selectedExpansionSetId} class="st-select w-full">
+        <div class="sne-expansion-set-select min-w-28">
```

Comment:
Empty state message is getting cut off slightly

<img width="166" height="69" alt="Image" src="https://github.com/user-attachments/assets/4c14f34e-0011-4956-a26b-1f8c63996aea" />